<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestionvol</class>
 <widget class="QMainWindow" name="gestionvol">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SKYUP AIRLINE</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 85, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="mainFrame">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>631</width>
      <height>531</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Modifier">
     <attribute name="title">
      <string>Ajouter un vol</string>
     </attribute>
     <widget class="QDateEdit" name="dateDepartVol">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>200</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>320</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Heure de arrivée du vol :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Date d'arrivée du vol :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Heure de départ du vol :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifierButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>470</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Valider </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prixVol">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>160</y>
        <width>351</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateArriveeVol">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>240</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTimeEdit" name="heureArriveeVol">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>320</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="nbPlaceReserve">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>120</y>
        <width>351</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTimeEdit" name="heureDepartVol">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>280</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Prix du vol  :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre de places réservées :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Durée du vol :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dureeVol">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>421</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Date de départ du vol :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 85, 255);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Ajouter un vol &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="numeroVol">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>421</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Numéro de vol :</string>
      </property>
     </widget>
     <widget class="QLabel" name="testAjout">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>420</y>
        <width>461</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>350</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom airport Arrivee:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="airportArrivee">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>350</y>
        <width>351</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="airportDepart">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>380</y>
        <width>351</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>380</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom airport Depart:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="SupprimerVol">
     <attribute name="title">
      <string>Supprimer un vol</string>
     </attribute>
     <widget class="QPushButton" name="supprimerButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>400</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Valider </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 85, 255);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Supprimer un vol &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="numVolSupp">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>421</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Numéro de vol :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Les informations du vol:</string>
      </property>
     </widget>
     <widget class="QLabel" name="testSupp">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>531</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_supp">
      <property name="geometry">
       <rect>
        <x>45</x>
        <y>181</y>
        <width>531</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ConsulterVol">
     <attribute name="title">
      <string>Consulter les vols</string>
     </attribute>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 85, 255);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Consulter la liste des  vols &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_ConsulterVol">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>581</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; text-decoration: underline; color:#ffffff;&quot;&gt;Gestion des vols&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; text-decoration: underline; color:#ffffff;&quot;&gt;ns &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Retour</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
